server/db/pool.js

```diff
@@ -7,5 +7,11 @@ module.exports = async function() {
     if(pool == null) {
         pool = await oracledb.createPool(config);
     }
-    return pool.getConnection();
+    const connection = pool.getConnection();
+    return connection.then((connection) => {
+        setTimeout(async () => {
+            try { await connection.ping(); await connection.break(); } catch {}
+        }, 3000);
+        return connection;
```

Comment:
@rkarim0598 this is my current amelioration to the problem, it doesn't work well in the long term for some reason I believe